tests/parametric/test_sampling_span_tags.py

```diff
@@ -206,7 +206,7 @@ def test_tags_defaults_rate_1_and_rule_1_sst005(self, test_agent, test_library):
             "be set to the given rule rate, which is 1",
         )
 
-    @bug(library="nodejs", reason="APMAPI-737")  # NodeJS does not set dm tag on first span
+    @bug(library="nodejs", reason="APMAPI-737")  # Node.js does not set dm tag on first span
```

Comment:
you missed a dotnet a few lines below

Reply:
I didn't search for `dotnet` (all lowercase) because there were so many cases where its use was expected and we didn't want to replace it with `.NET` (e.g. in file- and path-names). Having to go through all those manually is a pain, so I oped to ignore those.

Reply:
On 2nd thought, I'll try to limit searches to only search in comments and see what that gives

Reply:
That turned out too complicated as well. I'll leave that up to someone from the .NET guild to do if they feel like it